src/rdgz.c

```diff
@@ -57,13 +58,17 @@ void *rd_gz_decompress (const void *compressed, int compressed_len,
 		int r;
 		
 		if ((r = inflateInit2(&strm, 15+32)) != Z_OK)
+		{
+			rd_rkb_log(rkb, LOG_ERR, "GZIP", "inflateInit2 failed %d",  r);
```

Comment:
This error message is not very useful for the use which would need to check the Z_ERROR_.. defines to know what the error was.
Translate the error code to a string.